code/game/objects/effects/decals/Cleanable/humans.dm

```diff
@@ -34,12 +36,13 @@ var/global/list/image/splatter_cache=list()
 				mode_ticker.blood_check()
 	if(istype(src, /obj/effect/decal/cleanable/blood/gibs))
 		return
-	if(src.type == /obj/effect/decal/cleanable/blood)
+	if(istype(src, /obj/effect/decal/cleanable/blood) || istype(src, /obj/effect/decal/cleanable/blood/splatter))
```

Comment:
@tigercat2000 and besides the redundent check what else is off about the code and how could i improve?